React/src/screens/Dashboard/screens/Home/index.tsx

```diff
@@ -1,70 +1,47 @@
-import { useTranslation } from 'react-i18next';
 import { useForm } from 'react-hook-form';
 
-import { actionCreators as authActions } from 'contexts/UserContext/reducer';
-import { useDispatch as useUserDispatch } from 'contexts/UserContext';
-import { logout, removeCurrentUserToken } from 'services/AuthService';
-import FormInput from 'components/FormInput';
+import { UserData } from 'interfaces/userData';
 
-import logo from './assets/logo.svg';
-import styles from './styles.module.scss';
-import { withContextProvider, useSelector, useDispatch } from './context';
-import { actionCreators } from './context/reducer';
+import LogoWolox from '../../../../resources/img/logoWolox.png';
 
-interface TechForm {
-  tech: string;
-}
+import styles from './styles.module.scss';
+import { withContextProvider } from './context';
 
 function Home() {
-  const { t, i18n } = useTranslation();
-
-  // Example of how to use these custom hooks
-  const tech = useSelector((state) => state.tech);
-  const dispatch = useDispatch();
-  const userDispatch = useUserDispatch();
-  const { register, handleSubmit } = useForm<TechForm>();
-
-  const handleLogout = async () => {
-    await logout();
-    userDispatch(authActions.resetUser());
-    removeCurrentUserToken();
-  };
-
-  const handleChangeLanguage = () => {
-    i18n.changeLanguage(i18n.language === 'es' ? 'en' : 'es');
-  };
-
-  const onSubmit = handleSubmit((values) => {
-    if (values.tech) {
-      dispatch(actionCreators.setTech(values.tech));
-    }
-  });
+  const { register, handleSubmit } = useForm<UserData>();
 
   return (
-    <div className={styles.app}>
-      <header className={styles.appHeader}>
-        <img src={logo} className={styles.appLogo} alt="logo" />
-        <p className={styles.text}>{t('Home:loggedIn')}</p>
-        <p className={styles.text}>{t('Home:techIs', { tech })}</p>
-        <form className="column center m-bottom-10" onSubmit={onSubmit}>
-          <FormInput
-            className="m-bottom-2"
-            placeholder={t('Home:newTech')}
-            inputRef={register('tech')}
-            name="tech"
-            inputType="text"
-          />
-          <button className={styles.appLink} type="submit">
-            {t('Home:setNewTech')}
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
+        <img src={LogoWolox} alt="Wolox Logo Image" />
+        <form
+          onSubmit={handleSubmit((data) => console.log({ users: data }))}
```

Comment:
Little typo here. Also keep in mind that here we are returning a `console.log()`. It will work as expected, but we need to more declarative with our intent. Because in this case we would be returning and executing "console.log()" and we only want to execute it.

```suggestion
onSubmit={handleSubmit((data) => {
  console.log({ user: data })
  }
)}
```